scripts/collect_images.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+import datetime
+import json
+import os
+import subprocess
+
+from src.db.db_connection import sql_cursor
+
+"""
+This script should be set up with a cron job to run daily.
+rclone should be set up, in our case pointing to a Google Drive folder: https://rclone.org/drive/
+
+Collect images of non-goggle detections from the database.
+Upload images and metadata to Google Drive.
+"""
+
+METADATA_FILE = 'metadata.json'
+# rclone on ee220clnx1 is an earlier version that doesn't support copying to shared folders
+RCLONE_PATH = '/home/shay/a/bergz/rclone-v1.52.2-linux-amd64/rclone'
```

Comment:
I wasn't sure if this should be an argument or not, but there's no reason it would ever change. I made this executable able to be executed by anyone on the machine, so this script could be run by anyone.

Reply:
yeah lets make it an argument with a default value.